tests/integration/protostar_fixture.py

```diff
@@ -257,6 +258,7 @@ async def invoke(
         function_name: str,
         inputs: Optional[CairoOrPythonData],
         gateway_url: str,
+        resolve_proxies: bool = False,
```

Comment:
It looks like `resolve_proxies` isn't used by the command.

Reply:
right 👍